### DISCUSSION TITLE
Er...

### DISCUSSION THREAD

**Discussion by westnordost** [streetcomplete/StreetComplete#5573] [Ideas]
Hey, how about we change the wording of "Other Answers..." to 'Er..." or "Uh..."? It's not that awkwardly long (especially in some languages) and more or less has the same meaning. I.e. users should tap it if the answers presented don't seem to be covering it, it can't be answered or the question itself is moot.

**Comment by FloEdelmann** [comment]
After thinking about it a bit, I actually like the idea. It fits the playful style of the app, is much quicker to parse than "Other answers…" and is more inviting to try and tap it and thus makes it easier to understand the feature.

**Comment by westnordost** [comment]
https://books.google.com/ngrams/graph?content=er...%2Cuh...%2Cerm...%2Cuhm...&year_start=1969&year_end=2019&corpus=en-US-2019&smoothing=0&case_insensitive=true

**Reply by FloEdelmann** [reply]
Hmm... https://books.google.com/ngrams/graph?content=er...%2Cuh...%2Cerm...%2Cuhm...%2Chm...%2Chmm...%2Chmmm...&year_start=1969&year_end=2019&corpus=en-US-2019&smoothing=0&case_insensitive=true

**Comment by mnalis** [comment]
Is there a specific technical reason (e.g. trying to fit it in some smaller button or something), or just for trying to be less formal / more fun?

If only the latter, I see no problem, as at worst translators can keep their current text if they can't find better one - there could be complicated nuances in those... But if it must be short however for some technical reason, that might turn out to be problematic.

Another alternative might be `huh?` or `what?`, or _"confused"_ emoji (although that one might bring more confusion than it solves)...

**Reply by westnordost** [reply]
Just less formal, no technical reason.

**Comment by matkoniecz** [comment]
For me at least it would be quite confusing. Though I would learn quickly even if I would not be aware of this discussion.

I liked idea of "Other..." that was proposed by someone at US Slack.

**Comment by derfasthirnlosenick** [comment]
Having received this in the beta update, I can confirm it was quite confusing. I prefer "Other answers" or "other".

If you want to go the "uhh..." route, at least add the shrug emoji to get the point across to dummies like me

🤷 or ¯\\\_(ツ)\_/¯

**Comment by FloEdelmann** [comment]
For reference: This was implemented in https://github.com/streetcomplete/StreetComplete/commit/f7e0c5c0499d638e8e8f160552b7f691d8ae0be2

**Comment by mnalis** [comment]
ref problems with it: https://github.com/streetcomplete/StreetComplete/issues/5630, https://github.com/streetcomplete/StreetComplete/issues/6207
(I seem to recall other one about being confusing some weeks ago; but can't find it now)

**Comment by gdabski** [comment]
I was sure this is some sort of truncated **ER**ROR message, lol. But I don't mind now that I know.
